### ISSUE TITLE
Indicate focus to improve accessibility

### ISSUE THREAD

**Issue by metamoni** [rubyforgood/mutual-aid#776]
At the moment it's very difficult to tell when a button's been focused (see image below). Updated styles across the app to indicate focus and improve accessibility.

<img width="429" alt="Screenshot 2020-10-12 at 08 50 36" src="https://user-images.githubusercontent.com/22390758/95719801-41806e00-0c68-11eb-8f6d-636b1a15398e.png">

**Comment by natastro** [comment]
I'd be happy to look into this and help!

**Comment by exbinary** [comment]
That would be great @natastro 🙏🏾

We're using [bulma](bulma.io) and [buefy](buefy.org) with a few customizations in:
* [bulma_and_buefy.scss](../blob/main/app/assets/stylesheets/bulma_and_buefy.scss), and
* [bulma_simple_form_customization.scss](../blob/main/app/assets/stylesheets/bulma_simple_form_customization.scss)

I don't _think_ we've overriden anything that would impact how focus appears, so i'd first check if this is an issue with bulma itself (out of the box).

**Comment by natastro** [comment]
Hi @metamoni and @exbinary !
Below is a screenshot of a suggestion to use for improving focus. I do want to note, the button text and button color do not meet contrast requirements which may need to be addressed along with the focus. 

Text: #FFFFFF, Button color: #00D1B2, The contrast ratio is: 2:1
One thing to do could be to change the text color to black or #4A4A4A which is used throughout the site. Another option is to keep the white text and change the blue colored button to something like #008667 
 - [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)


![image](https://user-images.githubusercontent.com/17433156/97518576-dcb65a80-196d-11eb-8fdc-e0d6cc385b4e.png)

Looking forward to hearing from you!

**Comment by metamoni** [comment]
Looks much better, @natastro!
